releasetasks/release_configs/dev_mozilla-release_fennec_rc_graph_2.yml

```diff
@@ -23,24 +23,23 @@ update_verify_enabled: false
 push_to_releases_enabled: true
 # rc graph 2 only needs final_verify_channels
 channels:
-publish_to_balrog_channels:
-    - "release-dev"
+publish_to_balrog_channels: []
 final_verify_channels:
     - "release-dev"
-postrelease_bouncer_aliases_enabled: true
-uptake_monitoring_enabled: true
-postrelease_version_bump_enabled: true
-postrelease_mark_as_shipped_enabled: true
+postrelease_bouncer_aliases_enabled: false
+uptake_monitoring_enabled: false
+postrelease_version_bump_enabled: false
+postrelease_mark_as_shipped_enabled: false
 
-product: "fennec"
+product: "mobile"
```

Comment:
Hmmm, we don't usually use "mobile" for product, it's either stage_product or appName, see https://hg.mozilla.org/build/buildbot-configs/file/tip/mozilla/release-fennec-mozilla-release.py.template#l19. I'm not sure where it is used in the graph, but I'd check it. This also applies to the file renames.